frontend/utilities/constants.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
I think we should use disabling the one rule that's broken in this file, which is no-useless-escapes. I wouldn't want to push another bug because we disabled all linting on this file since we modify this file pretty frequently.

The one liner for that is:
` /* eslint no-useless-escape: 0 */`

Reply:
Good point. Fixed!